### PR TITLE
Markdown content upload

### PR DESCRIPTION
**Pull request by jakecosgrove**
## Description

All markdown pages added and populated with static content. 

Pages do not have:
- internal or external links
- inset text components
- tables
- images with captions

## Type of changes

- [x] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: docs/pages/features/text-messages.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Use NHS Notify to bulk send text messages such as invitations, alerts, notifications or reminders to your audiences.
+
+[insert image with caption here]
+
+## What you need to know
+
+You might want to understand more about:
+
+- [text message pricing](/pricing/text-messages)
```
Comment (RossBugginsNHS) [inline docs/pages/features/text-messages.md:20]: link format

File: docs/pages/get-started/get-started.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+permalink: /get-started/
+---
+
+## 1. Check if NHS Notify is right for you
+
+You can send messages with NHS Notify if your organisation or service is part of NHS England and supports direct care.
+
+You might want to learn more about:
+
+- [our features](/features)
```
Comment (RossBugginsNHS) [inline docs/pages/get-started/get-started.md:18]: links

File: docs/pages/pricing/pricing.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Find out [how to pay](/pricing/how-to-pay).
+
+## Text messages (SMS)
+
+NHS England organisations and service have an annual allowance of free text messages.
+
+When you've used your annual allowance, it costs 2.27 pence (plus VAT) per text message.
+
+[More about text message pricing](/pricing/text-messages)
```
Comment (RossBugginsNHS) [inline docs/pages/pricing/pricing.md:28]: links

File: docs/pages/using-nhs-notify/api.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+layout: page
+title: API
+parent: Using NHS Notify
+nav_order: 1
+permalink: /using-nhs-notify/api
+---
+
+NHS England organisations and services can integrate the NHS Notify API with their web applications or back office systems.
+
+<a href="https://digital.nhs.uk/developer/api-catalogue/nhs-notify" target="_blank">Read our API documentation (opens in a new tab)</a>.
```
Comment (RossBugginsNHS) [inline docs/pages/using-nhs-notify/api.md:14]: Use a markdown url for this, not href

**Review by RossBugginsNHS — CHANGES_REQUESTED** [review]

File: docs/pages/features/text-messages.md (hunk 1 above)
Comment (jakecosgrove) [reply docs/pages/features/text-messages.md:20]: Good spot cheers!

File: docs/pages/get-started/get-started.md (hunk 2 above)
Comment (jakecosgrove) [reply docs/pages/get-started/get-started.md:18]: Good spot cheers!

File: docs/pages/pricing/pricing.md (hunk 3 above)
Comment (jakecosgrove) [reply docs/pages/pricing/pricing.md:28]: Good spot cheers!

File: docs/pages/using-nhs-notify/api.md (hunk 4 above)
Comment (jakecosgrove) [reply docs/pages/using-nhs-notify/api.md:14]: Replaced with Markdown equivalent - noob question, why is using in-line HTML bad practice in Markdown?

File: docs/pages/features/features.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+## See how your messages perform
+
+Track how many messages you’ve sent and find out which ones are not being delivered.
+
+## Automatic sending
+
+To send messages automatically, you’ll need a developer to integrate with the NHS Notify API.
+
+<a href="https://digital.nhs.uk/developer/api-catalogue/nhs-notify" target="_blank">Read our API documentation (opens in a new tab)</a>.
```
Comment (bhansell1) [inline docs/pages/features/features.md:65]: This looks like the last in-line html link

**Review by bhansell1 — APPROVED** [review]

**Review by RossBugginsNHS — APPROVED** [review]

**Review by RossBugginsNHS — APPROVED** [review]

**Review by RossBugginsNHS — APPROVED** [review]

**Review by bhansell1 — APPROVED** [review]

**Pull request merged**